extensions/machine-learning/src/views/models/manageModels/currentModelsTable.ts

```diff
@@ -152,7 +152,7 @@ export class CurrentModelsTable extends ModelViewBase implements IDataComponent<
 			);
 		}
 		this._table = modelBuilder.declarativeTable()
-			.withProperties<azdata.DeclarativeTableProperties>(
+			.withProps(
```

Comment:
Did you intend to change things outside the arc extension?

Reply:
Yup